docs/fledge-rule-DataAvailability/index.rst

```diff
@@ -0,0 +1,15 @@
+.. Images
+.. |data-availability| image:: images/data-availability.png
+
+DataAvailability Rule
+=====================
+
+This is built in rule that triggers every time it receives data that matches the asset code or audit code is given.
```

Comment:
This is a built in rule that triggers every time it receives data that matches an asset code or audit code that matches those given in the configuration.